quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/HelpCommandDistTest.java

```diff
@@ -154,8 +154,10 @@ private void assertSingleJvmStarted(CLIResult cliResult) {
     }
 
     private void assertHelp(CLIResult cliResult) {
-        // normalize the output to prevent changes around the feature toggles to mark the output to differ
-        String output = cliResult.getOutput().replaceAll("((Disables|Enables) a set of one or more features. Possible values are: )[^.]{30,}", "$1<...>");
+        // normalize the output to prevent changes around the feature toggles or events to mark the output to differ
+        String output = cliResult.getOutput()
+                .replaceAll("((Disables|Enables) a set of one or more features. Possible values are: )[^.]{30,}", "$1<...>")
+                .replaceAll("(create a metric. Possible values are:)[^.]{30,}.(Available|only|when|user|event|metrics|are|enabled.| )*", "$1<...>");
```

Comment:
Is the text  `Available only when user event metrics are enabled.` removed from the test on purpose? To me it seems it should be left there. 

This works well: https://github.com/mhajas/keycloak/commit/d3449e16db7e6dba3340a9f591193dad032147fb#diff-b2e06ccab46f581ecf9c69f2d760f6d060aacf51c0ff71d7fd88c11c979a4bf4R252

Reply:
I found that when adding and removing new event types, the word wrap of the text "Available only when user event metrics are enabled." changes. Once the word wrap changes, the test will fail. 

So to have a stable test even if events are added or removed, I found that ignoring the words was straightforward for me. I could add comment to explain the intent of this, as I don't have a different solution. WDYT?

Reply:
Hmm. I don't see any other solution either. Let's proceed with your solution then.